File: persona.py
class Persona:
    def __init__(self, nombre, edad, ciudad, telefono, ocupacion):
        self.nombre = nombre
        self.edad = edad
        self.ciudad = ciudad
        self.telefono = telefono
        self.ocupacion = ocupacion

    def get_nombre(self):
        return self.nombre

    def set_nombre(self, nombre):
        self.nombre = nombre

    def get_edad(self):
        return self.edad

    def set_edad(self, edad):
        self.edad = edad

    def get_ciudad(self):
        return self.ciudad

    def set_ciudad(self, ciudad):
        self.ciudad = ciudad

    def get_telefono(self):
        return self.telefono

    def set_telefono(self, telefono):
        self.telefono = telefono

    def get_ocupacion(self):
        return self.ocupacion

    def set_ocupacion(self, ocupacion):
        self.ocupacion = ocupacion

    def __str__(self):
        return f"Persona(nombre='{self.nombre}', edad={self.edad}, ciudad='{self.ciudad}', telefono='{self.telefono}', ocupacion='{self.ocupacion}')"


if __name__ == "__main__":
    persona1 = Persona("Juan", 30, "Ciudad X", "123456789", "Ingeniero")

    print("Nombre:", persona1.get_nombre())
    print("Edad:", persona1.get_edad())
    print("Ciudad:", persona1.get_ciudad())
    print("Teléfono:", persona1.get_telefono())
    print("Ocupación:", persona1.get_ocupacion())

    persona1.set_edad(35)
    print("Nueva edad:", persona1.get_edad())

    print(persona1)

File: persona.java
public class Main {
    public static void main(String[] args) {
        Persona persona1 = new Persona("Juan", 30, "Masculino", "123 Calle Principal", "Ingeniero");

        System.out.println("Nombre: " + persona1.getNombre());
        System.out.println("Edad: " + persona1.getEdad());
        System.out.println("Género: " + persona1.getGenero());
        System.out.println("Dirección: " + persona1.getDireccion());
        System.out.println("Ocupación: " + persona1.getOcupacion());

        persona1.setEdad(35);
        System.out.println("Nueva edad: " + persona1.getEdad());

        System.out.println(persona1);
    }
}

class Persona {
    private String nombre;
    private int edad;
    private String genero;
    private String direccion;
    private String ocupacion;

    public Persona(String nombre, int edad, String genero, String direccion, String ocupacion) {
        this.nombre = nombre;
        this.edad = edad;
        this.genero = genero;
        this.direccion = direccion;
        this.ocupacion = ocupacion;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public int getEdad() {
        return edad;
    }

    public void setEdad(int edad) {
        this.edad = edad;
    }

    public String getGenero() {
        return genero;
    }

    public void setGenero(String genero) {
        this.genero = genero;
    }

    public String getDireccion() {
        return direccion;
    }

    public void setDireccion(String direccion) {
        this.direccion = direccion;
    }

    public String getOcupacion() {
        return ocupacion;
    }

    public void setOcupacion(String ocupacion) {
        this.ocupacion = ocupacion;
    }

    @Override
    public String toString() {
        return "Persona{" +
                "nombre='" + nombre + '\'' +
                ", edad=" + edad +
                ", genero='" + genero + '\'' +
                ", direccion='" + direccion + '\'' +
                ", ocupacion='" + ocupacion + '\'' +
                '}';
    }
}
